index.html

```diff
@@ -86,18 +86,21 @@ <h3>An Open-Source Conversational AI Toolkit</h3>
 								<a class="banner_btn" href="tutorial_basics.html">Get Started</a>
 								<a class="banner_btn2" href="https://github.com/speechbrain/speechbrain">GitHub</a>
 								<a class="banner_btn4" href="https://huggingface.co/speechbrain"><img src="img/hf.svg" style="width:40px"/></a>
-								<iframe src="https://ghbtns.com/github-btn.html?user=speechbrain&repo=speechbrain&type=star&count=true&size=large&v=2" frameborder="0" scrolling="0" width="170" height="30" title="GitHub"></iframe>
-                                <a href="https://discord.gg/3wYvAaz3Ck"><img src="https://dcbadge.vercel.app/api/server/3wYvAaz3Ck?style=flat" /></a>
-		<br><br> <a href="https://drive.google.com/file/d/1Ig9KRzQ--H-0ugmz2UXA_pdxTuLBGhtk/view?usp=sharing""><font size="+1"> The call for Sponsors 2023 is open!</font></a>

```

Comment:
why did you remove this link ?

Reply:
@Adel-Moumen restored, and i took all new screenshots